addons/viewport/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@storybook/addon-viewport",
+  "version": "3.2.0",
+  "description": "Storybook addon to change the viewport size to mobile",
+  "main": "dist/index.js",
+  "keywords": [
+    "storybook"
+  ],
+  "scripts": {
+    "build": "babel --source-maps --out-dir dist src",
+    "watch": "npm run build -- -w",
+    "prepublish": "npm run build && node ../../scripts/prepublish.js",
+    "storybook": "start-storybook -p 3000",
+    "deploy-storybook": "storybook-to-ghpages"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "@storybook/react": "^3.2.5",
+    "prop-types": "^15.5.10"
+  },
+  "peerDependencies": {
+    "@storybook/addons": "^3.2.0",
+    "react": "*"
+  },
+  "devDependencies": {
+    "@storybook/addons": "^3.2.0",
+    "babel-cli": "^6.24.1",
```

Comment:
I think all the devDependencies can be removed.

should be using babel-preset-env if babel preset need to stay for some reason.